modules/Core/includes/endpoints/InfoEndpoint.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * No params
+ * 
+ * @return string JSON Array of NamelessMC information
+ */
+class InfoEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'info';
+        $this->_module = 'Core';
+        $this->_description = 'Return info about the Nameless installation';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.